components/page-end-cta.jsx

```diff
@@ -0,0 +1,17 @@
+var React = require('react');
+var Router = require('react-router');
+var Link = Router.Link;
+
+var PageEndCTA = React.createClass({
+  render: function() {
+    return (
+      <div className="page-end-cta">
+        <p>{this.props.description}</p>
```

Comment:
Oops, didn't realize Aki's fix has been landed already.  I see
```
html, body {
  font-size: 62.5%;
}
```

I'll go through this file and fix units.